README.md

```diff
@@ -36,6 +50,9 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+1. We propose smart shock collars with real time internet activity, vital metrics and body camera monitoring. The collar will track desktop and mobile usage. The user can specify which activities they would like to avoid, and the time period for which they would like to avoid said activities. When a user engages in the specified activities during the specified time period, the user will receive a minor electrical shock that will not stop until the activity is stopped.
```

Comment:
The interesting thing here is that a user would willingly by the item in response to bad habits. One has to wonder if ethics comes into play (i.e. is it ethical to offer a product like this on the market?), as I would expect many to object and have a desire to have a more user-friendly solution.